### DISCUSSION TITLE
Update IDH from upstream

### DISCUSSION THREAD

**Discussion by bergsjoh** [Security-Onion-Solutions/securityonion#13056] [2.4] [answered]
### Version

2.4.60

### Installation Method

Security Onion ISO image

### Description

upgrading

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

*

### RAM

*

### Storage for /

*

### Storage for /nsm

*

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Would be nice to have a refreshed version of opencanary in so-idh.
There are several fixes implemented and features added, [ssh kex parts bug](https://github.com/thinkst/opencanary/issues/186) and [https module](https://github.com/thinkst/opencanary/pull/221) to mention a couple.

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by defensivedepth** [comment, marked as answer]
We check for / install the latest version every release. The latest release from OpenCanary is from March, 2024 - https://github.com/thinkst/opencanary/releases/tag/v0.9.3
